src/components/bookInformation.tsx

```diff
@@ -8,7 +8,7 @@ type Props = {
   setList: (val: any[]) => void;
 };
 
-const BookInformation: React.FC<Props> = (props) => {
+function BookInformation({ isbn, list, setIsbn, setList }: Props) {
```

Comment:
What's the difference here

Reply:
Honestly I just didn't understand the necessity for `React.FC<Props>` so I made it more readable for myself.